.circleci/config.yml

```diff
@@ -262,11 +262,31 @@ jobs:
         steps:
             - prepare-rtl:
                 project-key: "chipyard-ariane"
-    prepare-chipyard-sodor:
+    prepare-chipyard-sodor-stage1:
         executor: main-env
         steps:
             - prepare-rtl:
-                project-key: "chipyard-sodor"
+                project-key: "chipyard-sodor-stage1"
+    prepare-chipyard-sodor-stage2:
+        executor: main-env
+        steps:
+            - prepare-rtl:
+                project-key: "chipyard-sodor-stage2"
+    prepare-chipyard-sodor-stage3:
+        executor: main-env
```

Comment:
This is too many tests, it will slow down CI too much. Just test the stage5 design.